server.go

```diff
@@ -463,15 +464,22 @@ func (s *Server) GetClient(w http.ResponseWriter, r *http.Request, ps httprouter
 		dns = fmt.Sprint("DNS = ", *wgDNS)
 	}
 
+	keepalive := ""
+	if *wgKeepAlive != "" {
+		keepalive = fmt.Sprint("PersistentKeepalive = ", *wgKeepAlive)
+	}
+
 	configData := fmt.Sprintf(`[Interface]
-%s
 Address = %s
 PrivateKey = %s
+%s
+
 [Peer]
 PublicKey = %s
 AllowedIPs = %s
 Endpoint = %s
-`, dns, client.IP.String(), client.PrivateKey, s.Config.PublicKey, allowedIPs, *wgEndpoint)
+%s
```

Comment:
I think it could be nice to have:
```
PersistentKeepalive = %s
```
as opposed to just `%s` to keep it consistent and it's also easier to understand what the final config will look like.

Reply:
It needs to have a value when it's in the config, that's why much like DNS it's added only when specified.

Reply:
Ah ok! Nevermind me then.